daemon/ucrednet.go

```diff
@@ -116,14 +116,18 @@ func (wl *ucrednetListener) Accept() (net.Conn, error) {
 
 	var unet *ucrednet
 	if ucon, ok := con.(*net.UnixConn); ok {
-		f, err := ucon.File()
+		syscallConn, err := ucon.SyscallConn()
 		if err != nil {
 			return nil, err
 		}
-		// File() is a dup(); needs closing
-		defer f.Close()
```

Comment:
so was the issue essentially that this defer would never get executed because we are blocked inside Accept forever?

Reply:
Finally found out the issue: it's the call to File.Fd() which breaks everything. I added a paragraph to the PR description (and to one of the commits).